R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -906,6 +910,37 @@ predicted <- predict(model, df)
 head(predicted)
 ```
 
+#### FP-growth
+
+`spark.fpGrowth` executes FP-growth algorithm to mine frequent itemsets on a `SparkDataFrame`. `itemsCol` should be an array of values.
+
+```{r}
+items <- selectExpr(createDataFrame(data.frame(items = c(
+  "T,R,U", "T,S", "V,R", "R,U,T,V", "R,S", "V,S,U", "U,R", "S,T", "V,R", "V,U,S",
+  "T,V,U", "R,V", "T,S", "T,S", "S,T", "S,U", "T,R", "V,R", "S,V", "T,S,U"
+))), "split(items, ',') AS items")
```

Comment:
that's possible but I'm fairly certain there are still quite a few functions we have missed over the year that are not in JIRA.

feel free to add them - would appreciate your help. 

agree `array` is a bit tricky - I'd rather not having to diverge because of consistency with `array_contain` function and so on, but I can see `spark.array` might be an approach. Or perhaps `array_col`?